### ISSUE TITLE
Cannot send to "short codes" -- like 71730

### ISSUE THREAD

**Issue by skandragon** [tulak/pdu_tools#7]
I have the need to send to a thing that is not a proper phone number.  However, it looks like there is no way to do this currently:

```
`initialize': Must enter area code or set default area code (Phoner::AreaCodeError)
```

I may take a stab at this later, probably with a "I know what I'm doing, just let me do it" flag to skip sanitizing the phone number field.

**Comment by skandragon** [comment]
I also rolled this into my MegaPullRequest #6

**Comment by tulak** [comment]
Solved by #6
